<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewsList</class>
 <widget class="QWidget" name="NewsList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>913</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>News</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/resource/16x16_ico_png/ico_news.png</normaloff>:/resource/16x16_ico_png/ico_news.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>News</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QComboBox" name="cmbAccounts">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkFavorites">
          <property name="text">
           <string>Show favorites only</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="tableGroups">
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>0</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>16</number>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>16</number>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>12</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="value">
         <number>24</number>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/16x16_ico_png/ico_refresh.png</normaloff>:/resource/16x16_ico_png/ico_refresh.png</iconset>
   </property>
   <property name="text">
    <string>Reresh</string>
   </property>
   <property name="toolTip">
    <string>Refresh the list of newsgroups</string>
   </property>
  </action>
  <action name="actionFavorite">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/16x16_ico_png/ico_star.png</normaloff>:/resource/16x16_ico_png/ico_star.png</iconset>
   </property>
   <property name="text">
    <string>Favorite</string>
   </property>
   <property name="toolTip">
    <string>Mark as favorite newsgroup</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/16x16_ico_png/ico_stop.png</normaloff>:/resource/16x16_ico_png/ico_stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionBrowse">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/16x16_ico_png/ico_news.png</normaloff>:/resource/16x16_ico_png/ico_news.png</iconset>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
   <property name="toolTip">
    <string>Browse the contents of the news group</string>
   </property>
  </action>
  <action name="actionUnfavorite">
   <property name="icon">
    <iconset>
     <normaloff>:/resource/16x16_ico_png/ico_star_gray.png</normaloff>:/resource/16x16_ico_png/ico_star_gray.png</iconset>
   </property>
   <property name="text">
    <string>Unfavorite</string>
   </property>
   <property name="toolTip">
    <string>Mark as nonfavorite</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
